modules/hcp-get-ocp-channel.adoc

```diff
@@ -2,134 +2,82 @@
 //
 // * hosted_control_planes/hcp-updating.adoc
 
-:_mod-docs-content-type: CONCEPT
+:_mod-docs-content-type: PROCEDURE
 [id="hcp-get-ocp-channel_{context}"]
-= Setting channels in a hosted cluster
+= Updating your hosted cluster
 
-You can see available updates in the `HostedCluster.Status` field of the `HostedCluster` custom resource (CR).
+You can update the hosted cluster by setting a channel in your `HostedCluster` resource or by using the `oc patch` command.
 
-The available updates are not fetched from the Cluster Version Operator (CVO) of a hosted cluster. The list of the available updates can be different from the available updates from the following fields of the `HostedCluster` custom resource (CR):
+The initial `HostedCluster` resource does not have any information in the `status.version.availableUpdates` and `status.version.conditionalUpdates` fields. After you set the `spec.channel` field to the stable {product-title} release version, the HyperShift Operator reconciles the `HostedCluster` resource and updates the `status.version` field with the available and conditional updates.
 
-* `status.version.availableUpdates`
-* `status.version.conditionalUpdates`
+.Procedure
 
-The initial `HostedCluster` CR does not have any information in the `status.version.availableUpdates` and `status.version.conditionalUpdates` fields. After you set the `spec.channel` field to the stable {product-title} release version, the HyperShift Operator reconciles the `HostedCluster` CR and updates the `status.version` field with the available and conditional updates.
+. Check recommended {product-title} channels by running the following command:
++
+[source,terminal]
+----
+$ oc -n <hosted_cluster_namespace> get -o jsonpath='{.status.version.desired.channels}' hostedcluster <hosted_cluster_name>
+----
 
-See the following example of the `HostedCluster` CR that contains the channel configuration:
+. Set a channel in your hosted cluster that matches with your desired channel semantics. For more information about channels and releases, see "Understanding update channels and releases". Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n <hosted_cluster_namespace> hostedcluster <hosted_cluster_name> --type=merge -p '{"spec":{"channel":"stable-<4.y>"}}' <1>
+----
+<1> Replace `<4.y>` with the {product-title} release version that you specified in `spec.release`. For example, if you set the `spec.release` to `ocp-release:4.18.4-multi`, you must set `spec.channel` to `stable-4.18`.
```

Comment:
you have [options][1] beyond `stable`, and also beyond strictly matching your current 4.y.  For example, `stable-4.18` doesn't include any 4.19 or later releases, and some of the folks currently running 4.18.z releases will eventually want to update to 4.19 releases.  Right now, 4.18.4 is in:

```console
$ curl -s 'https://api.openshift.com/api/upgrades_info/graph?channel=stable-4.18' | jq -r '.nodes[] | select(.version == "4.18.4").metadata["io.openshift.upgrades.graph.release.channels"]'
candidate-4.18,eus-4.18,fast-4.18,stable-4.18,candidate-4.19,fast-4.19,candidate-4.20
```
```

[1]: https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/updating_clusters/understanding-openshift-updates-1#understanding-update-channels-releases